utils/common/src/test/java/org/apache/brooklyn/util/javalang/MemoryUsageTrackerTest.java

```diff
@@ -82,7 +82,7 @@ private long sizeOfActiveReferences(List<Maybe<byte[]>> references) {
         return size;
     }
 
-    @Test(groups="Integration")
+    @Test(groups="Live")
```

Comment:
I'm currently setting it up so it's still unofficial. It's running in a docker container which lets us set the environment as needed.

Note the messages says `cleared after 1000001` which means that the first reference never got cleared, it just got to the end of the loop. Looking at all references sounds good.